pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -1191,9 +1191,9 @@ func checkCRIPSIStats(assert *assert.Assertions, want, got *cadvisorapiv1.PSISta
 
 func checkCRIPSIData(assert *assert.Assertions, want, got cadvisorapiv1.PSIData) {
 	assert.Equal(want.Total, got.Total)
-	assert.Equal(want.Avg10, got.Avg10)
-	assert.Equal(want.Avg60, got.Avg60)
-	assert.Equal(want.Avg300, got.Avg300)
+	assert.InDelta(want.Avg10, got.Avg10, 0.01)
```

Comment:
Can we fold these changes into the commit where the changes are first introduced?